tests/RequestBuilder/RequestBuilderFactoryTest.php

```diff
@@ -57,10 +60,16 @@ public function provideBuilderMethods(): array
             $builder->addItemProperty(ItemProperty::create('item-id', []));
         };
 
+        $sortingInit = function (SortingRequestBuilder $builder): void {
+            $builder->addUserMerge(UserMerge::mergeInto('user-a', 'user-b'));
```

Comment:
This should be just "minimal" setup to make the internal validation (eg. at leas one command in other cases) passes. As there is this kind of validation, this whole could be omitted.